sycl/test/basic_tests/vec_convert_half.cpp

```diff
@@ -0,0 +1,102 @@
+// XFAIL: cuda
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
+// %CPU_RUN_PLACEHOLDER %t.out
```

Comment:
Sorry, I had a typo in my previous suggestion. It should be:
```suggestion
// RUN: %CPU_RUN_PLACEHOLDER %t.out
```